plugin.py

```diff
@@ -20,6 +20,23 @@ class LspVolarPlugin(NpmClientHandler):
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', '@volar', 'server', 'out', 'index.js')
 
+    @classmethod
+    def on_client_configuration_ready(cls, configuration: dict) -> None:
+        is_lsp_typescript_enabled = sublime.load_settings("LSP-typescript.sublime-settings").get("enabled", False)
+        take_over_mode = configuration.get("settings", {}).get("volar.takeOverMode.enabled", "auto") # type: Union[str, bool]
+
+        def dont_start_in_typescript_files():
+            languages = configuration.get("languages", [])
+            languages_without_typescript = list(filter(lambda langDict: langDict.get('languageId') != 'typescript', languages))
+            configuration['languages'] = languages_without_typescript
+
+        if (take_over_mode == "auto" and is_lsp_typescript_enabled):
```

Comment:
Doing much JS programming? Parenthesis are not very python-like :)

I'd have a look and try the changes a bit later

Reply:
hahaha, yes :)

I tired to read the project configuration as well, to see if LPS-typescript is disabled in the project,
but that didn't worked in all cases... (I used `sublime.active_window()`, so it might happen that a wrong window is checked), so I reverted the commit.